src/components/_global/BalChart/BalChart.vue

```diff
@@ -69,7 +76,12 @@ type PeriodOption = {
 };
 
 export default defineComponent({
```

Comment:
Would be good to convert this component into using script setup instead of defineComponent. We've been slowly converting components over time.

Reply:
fixed